fetch-todolist/src/component/Home.js

```diff
@@ -29,29 +46,28 @@ const Home = () => {
   );
 
   const changeStatus = (check, id) => {
-    const updatedStatusArray = data.map((obj) => {
+    const object = data.find((obj) => {
       if (obj.id === id) {
+        console.log("id", id, "check ", check);
         obj.status = check;
       }
-      return obj;
     });
-    setTodos(updatedStatusArray);
+    dispatch({ type: ACTIONS.SET_TODOS, payload: data });
   };
 
   const { mutate: deleteTodoMutation } = useMutation(deleteTodo, {
     onSuccess: () => {
-      queryClient.invalidateQueries("todos");
+      queryClient.invalidateQueries(RQ_KEY_TODOS);
     },
   });
 
-  const handleDelete = (id) => {
-    deleteTodoMutation(id);
+  const handleDelete = () => {
+    deleteTodoMutation(idToDel);
```

Comment:
idToDel ? 
Try to avoid short forms

Reply:
i'll keep that in mind, and I've changed it i will push it with all changes